src/flippy.nim

```diff
@@ -870,3 +876,60 @@ proc resize*(srcImage: Image, width, height: int): Image =
       1
     ))
   )
+
+func width*(flippy: Flippy): int =
+  flippy.mipmaps[0].width
+
+func height*(flippy: Flippy): int =
+  flippy.mipmaps[0].height
+
+proc save*(flippy: Flippy, filePath: string) =
+  ## Flippy is a special file format that is fast to load and save with mip maps.
+  var f = newFileStream(filePath, fmWrite)
+  defer: f.close()
+
+  f.write(&"flippy!!0x{version.toHex(4)}\0")
```

Comment:
I think 4 bytes is enough for file format. I think thats how most do it. I would use "flip". 
I would write the version is an write(1.uint32) as another 4 bytes.

Reply:
done